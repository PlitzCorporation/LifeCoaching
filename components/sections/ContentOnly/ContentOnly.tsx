import React from 'react';

interface ContentOnlyProps {
	text: string;
	classes?: string;
}

const ContentOnly = (props: ContentOnlyProps) => {
	const { text } = props;
	return (
		<div className="bg-plitz-primary py-10 lg:py-20">
			<div
				dangerouslySetInnerHTML={{ __html: text }}
				className={`plitz-container text-balance whitespace-pre-wrap max-w-3xl ${props.classes}`}
			></div>
		</div>
	);
};

export default ContentOnly;
