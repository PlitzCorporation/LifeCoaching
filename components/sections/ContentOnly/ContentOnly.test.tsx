import React from 'react';
import { render } from '@testing-library/react';
import ContentOnly from './ContentOnly';

describe('ContentOnly', () => {
	it('renders component', () => {
		const text = '<p>Hello, I am Batman, but my real name is Bruce Wayne</p>';
		const classes = 'text-center text-white';
		const { container } = render(<ContentOnly text={text} classes={classes} />);
		expect(container.firstChild).toMatchSnapshot();
	});
});
