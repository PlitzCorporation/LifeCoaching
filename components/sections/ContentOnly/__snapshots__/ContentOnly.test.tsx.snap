// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentOnly renders component 1`] = `
<div
  class="bg-plitz-primary py-10 lg:py-20"
>
  <div
    class="plitz-container text-balance text-center text-white"
  >
    <p>
      Hello, I am Batman, but my real name is Bruce Wayne
    </p>
  </div>
</div>
`;
